packages/backend-core/tests/core/utilities/minio.ts

```diff
@@ -21,7 +22,8 @@ export async function start(): Promise<StartedTestContainer> {
     .withWaitStrategy(new ObjectStoreWaitStrategy().withStartupTimeout(30000))
     .start()
 
-  return container
+  const port = container.getMappedPort(9000)
+  env._set("MINIO_URL", `http://0.0.0.0:${port}`)
```

Comment:
This ip should come from the container. `container.host()` or something like this

Reply:
I'm actually in the process is explicitly _not_ getting this information from the container because I've found it can be incorrect. Some Docker implementations will bind a port on both IPv4 (127.0.0.1) and IPv6 (::1) localhost, but `container.getHost()` always gives you `localhost`. This can then resolve to either a v4 or v6 address. Then `container.getExposedPort(1234)` seems to usually give back the v6 address.

I have a PR I'm working on that always uses v4 for both things, and bypasses the `testcontainers` functions entirely.

Reply:
Yep, as Sam mentioned i did try to use the getHost() function but found it just didn't work.